plugins/ibm_qradar/requirements.txt

```diff
@@ -0,0 +1,5 @@
+# List third-party dependencies here, separated by newlines. 
+# All dependencies must be version-pinned, eg. requests==1.2.0
+# See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
+
+insightconnect-plugin-runtime
```

Comment:
```suggestion
```

This is unnecessary in the final requirements file. The [Dockerfile](https://github.com/rapid7/insightconnect-plugins/blob/a86b6875b216b8086a358ee4d4c4d652edd9ebde/plugins/ibm_qradar/Dockerfile#L1) that builds the production environment container that this plugin will ultimately run in already includes the InsightConnect plugin SDK in the base layer.

Reply:
Updated read me fil.